homework/index.js

```diff
@@ -0,0 +1,74 @@
+let repositories;
+async function getPromise() {
+  const res = await fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100');
+  return res.json();
+}
+
+const myPromise = getPromise(); // Set the promise as a global variable
+
+async function selectedRepoCaracteristics(selectValue) {
+  const result = await myPromise;
+  result.forEach((x) => {
+    if (x.name === selectValue) {
+      document.getElementById('repoCaracteristics').innerHTML = `
+         <b>Repository :</b>   <a href="">${x.name} </a><br>
+          <b>Descripion :</b>   ${x.description} <br>
+          <b>Forks :</b>   ${x.forks} <br>
+          <b>Updated :</b>   ${new Date(x.updated_at).toLocaleString('en-US')} <br>`;
+    }
+  });
+}
+
+
+function showContributors(list) {
+  // document.getElementById('repoContributors').removeChild();
+  let cont = '<ul class="list-group">';
+  list.forEach((item) => {
+    // cont += `<li>${item.avatar_url}</li>`
+    cont += `<li class="list-group-item"><a href="${item.html_url}" target="_blank">${item.login}</a></li>`;
+  });
+  cont += '</ul>';
+  document.getElementById('repoContributors').innerHTML = cont;
+}
+
+async function listcontributors(url) {
+  const response = await fetch(url);
+  const data = await response.json();
+  showContributors(data);
+}
+
+async function selectedRepoContributors(selectValue) {
```

Comment:
Your function name should reflect what it *does*. In this case, maybe `listContributorsOfSelectedRepo`? But you'll notice that this function actually doesn't do much. And if I'm not mistaken, in the next part of the homework, as fetching the contributors of a repo is done *in a method of the repo*, this code will move there ;).